Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM maven:3.8.4-jdk-11-slim as builder
+
+COPY common /usr/src/app/common
+COPY spring-boot-dixit /usr/src/app/spring-boot-dixit
+COPY dixit /usr/src/app/dixit
+COPY pom.xml /usr/src/app
+
+RUN mvn -f /usr/src/app/pom.xml clean package
```

Comment:
NIT: Can't it be skipped the test phrase?

Reply:
good advice